### ISSUE TITLE
bring in eval script

### ISSUE THREAD

**Issue by gngdb** [BayesWatch/tf-variational-dropout#4]
From the same tutorial as the rest of the cifar-10 training script: https://github.com/tensorflow/models/tree/master/tutorials/image/cifar10
